shotover-proxy/src/transforms/cassandra/sink_cluster.rs

```diff
@@ -31,29 +34,48 @@ impl CassandraSinkClusterConfig {
         Ok(Transforms::CassandraSinkCluster(CassandraSinkCluster::new(
             self.first_contact_points.clone(),
             chain_name,
+            self.data_center.clone(),
             tls,
         )))
     }
 }
 
 pub struct CassandraSinkCluster {
     contact_points: Vec<String>,
-    outbound: Option<CassandraConnection>,
+    contact_point_connection: Option<CassandraConnection>,
+    init_handshake: Vec<Message>,
+    init_handshake_address: Option<String>,
+    init_handshake_complete: bool,
     chain_name: String,
     failed_requests: Counter,
     tls: Option<TlsConnector>,
     pushed_messages_tx: Option<mpsc::UnboundedSender<Messages>>,
+    peer_table: FQName,
+    data_center: String,
+    nodes: Vec<CassandraNode>,
+    nodes_shared: Arc<RwLock<Vec<CassandraNode>>>,
```

Comment:
Are you still planning to remove the node sharing stuff and just send our own system.peers query?

Reply:
We do send our own system.peers query!
It gets shared between all the transforms through nodes_shared ... I should go rename nodes_shared now though.